Client/Application/LeanplumIntegration.swift

```diff
@@ -62,219 +62,176 @@ enum LeanplumEventName: String {
             return
         }
 
         switch settings.environment {
-        case .development:
-            log.info("LeanplumIntegration - Setting up for Development")
-            Leanplum.setDeviceId(UIDevice.current.identifierForVendor?.uuidString)
-            Leanplum.setAppId(settings.appId, withDevelopmentKey: settings.key)
-        case .production:
-            log.info("LeanplumIntegration - Setting up for Production")
-            Leanplum.setAppId(settings.appId, withProductionKey: settings.key)
-        }
-        Leanplum.syncResourcesAsync(true)
-
-        if profile?.prefs.boolForKey(PrefsKeys.HasFocusInstalled) == nil {
-            profile?.prefs.setBool(!canInstallFocus(), forKey: PrefsKeys.HasFocusInstalled)
+            case .development:
+                log.info("LeanplumIntegration - Setting up for Development")
+                Leanplum.setDeviceId(UIDevice.current.identifierForVendor?.uuidString)
+                Leanplum.setAppId(settings.appId, withDevelopmentKey: settings.key)
+            case .production:
+                log.info("LeanplumIntegration - Setting up for Production")
+                Leanplum.setAppId(settings.appId, withProductionKey: settings.key)
         }
 
-        if profile?.prefs.boolForKey(PrefsKeys.HasPocketInstalled) == nil {
-            profile?.prefs.setBool(!canInstallPocket(), forKey: PrefsKeys.HasPocketInstalled)
-        }
+        Leanplum.syncResourcesAsync(true)
 
-        var userAttributesDict = [AnyHashable: Any]()
-        userAttributesDict[UserAttributeKeyName.mailtoIsDefault.rawValue] = mailtoIsDefault()
-        userAttributesDict[UserAttributeKeyName.focusInstalled.rawValue] = !canInstallFocus()
-        userAttributesDict[UserAttributeKeyName.klarInstalled.rawValue] = !canInstallKlar()
-        userAttributesDict[UserAttributeKeyName.pocketInstalled.rawValue] = !canInstallPocket()
-        userAttributesDict[UserAttributeKeyName.signedInSync.rawValue] = profile?.hasAccount()
+        let attributes: [AnyHashable: Any] = [
+            LPAttributeKey.mailtoIsDefault: mailtoIsDefault(),
+            LPAttributeKey.focusInstalled: focusInstalled(),
+            LPAttributeKey.klarInstalled: klarInstalled(),
+            LPAttributeKey.pocketInstalled: pocketInstalled(),
+            LPAttributeKey.signedInSync: profile?.hasAccount() ?? false
+        ]
 
         self.setupCustomTemplates()
         
-        Leanplum.start(withUserId: nil, userAttributes: userAttributesDict, responseHandler: { _ in
-            self.track(eventName: LeanplumEventName.openedApp)
+        Leanplum.start(withUserId: nil, userAttributes: attributes, responseHandler: { _ in
+            self.track(event: .openedApp)
 
             // We need to check if the app is a clean install to use for
             // preventing the What's New URL from appearing.
-            if self.profile?.prefs.intForKey(IntroViewControllerSeenProfileKey) == nil {
+            if self.introSeen == nil {
```

Comment:
I think I meant `let prefs = self.profile?.prefs` here. 

If you're going to break this out into calculated properties, I'd go full bore setters as well, just to keep the getting and setting in the same place.